content/support-training.html

```diff
@@ -0,0 +1,132 @@
+---
+title: Support & Training
+layout: jumbotron
+---
+
+<div class="container-fluid training">
+  
+  <div class="row gray-bg">
+    <div class="container">
+    <div class="col-md-12 doc-content">
+      <h1>Support & Training</h1>
+      <p>This list is provided in alphabetical order. If you'd like to add your training resource, see <a href="#add">Add a resource</a>.
+    </div>
+    </div>
+  </div>
+
+
+  <div class="row">
+    <div class="container">
+      <div class="col-md-12 doc-content training-team">
+        <h2>Courses</h2>
+      </div>
+    </div>
+    <div class="container equal">
+      <div class="col-sm-12 col-md-6 doc-content panel-col">
+          <div class="panel panel-default">
+            <div class="panel-body">
+              <a href="https://training.robustperception.io/" target="_blank">
+                <h3>Learn how to Monitor with Prometheus</h3>
```

Comment:
This is still a marketing blurb.

The only text we should have here is the organisation names.